### DISCUSSION TITLE
Sciatic nerve toluidine blue image analysis on AxonDeepSeg

### DISCUSSION THREAD

**Discussion by apuer11** [neuropoly/axondeepseg#755] [Help needed]
Hi everyone!
I followed all of the steps to install AxonDeepSeg as a plugin on napari and everything seems to be set up correctly according to the quick check and running the sample images provided for the models. I am quite new to this, so any advice would be greatly appreciated. 
I am having trouble determining what are the steps I should take to prep my images to be analyzed using the pretrained models and which of the three would be best to use. I have seen some posts mentioning grayscale and cropping the images to a certain size, but I am not sure of exactly everything I should be doing for the model to easily analyze my image. I used ImageJ to determine the pixel size in microns and turn it into grayscale since I have not seen the original color stain images used by anyone. I was also wondering how I would determine the number of axons in the entire cross section if this tool only accepts/analyzes a small square of the whole image at a time? Moreover, I am not sure if the contrast of my images is not good enough and that is causing the problem because when I finally got it to run it took a while and did not create the red and blue layers distinguishing the myelin and axon. Thank you for your help in advance!
![Capture](https://github.com/axondeepseg/axondeepseg/assets/142545380/8082b059-3bd7-42e2-a34e-df5bc4cba258)

**Comment by Stoyan-I-A** [comment]
Hello @apuer11, I have responded in the issue you opened here: #754
